docs/network/net_binding_plugins/passt.md

```diff
@@ -114,13 +172,17 @@ specified in the Kubevirt CR when registering the network binding plugin.
 ### Feature Gate
 For KubeVirt versions prior to v1.5, make sure to enable the `NetworkBindingPlugins` FG.
 ```
-kubectl patch kubevirts -n kubevirt kubevirt --type=json -p='[{"op": "add", "path": "/spec/configuration/developerConfiguration/featureGates/-",   "value": "NetworkBindingPlugins"}]'
+kubectl patch kubevirt -n kubevirt kubevirt --type=merge -p='{"spec":{"configuration":{"developerConfiguration":{"featureGates":["NetworkBindingPlugins"]}}}}'
 ```
 
 > **Note**: The specific passt plugin has no FG by its own. It is up to the cluster
 > admin to decide if the plugin is to be available in the cluster.
 > The passt binding is still in evaluation, use it with care.
 
+**Note**:
+> It is up to the cluster admin to decide if the plugin is to be available in the cluster.
```

Comment:
I think that it is implicit since users have to manually deploy it on their clusters.

Reply:
hmm it is double there now, will update